EzMap.Domain/Repositories/PoiRepository.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System.Runtime.InteropServices.ComTypes;
+using Microsoft.EntityFrameworkCore;
+
+namespace EzMap.Domain.Repositories;
+
+public interface IPointOfInterests
```

Comment:
Please don't create the Repositories before you know what APIs are gonna be created.
Some of these won't even be used.
In the end no redundant code.

Reply:
Should only create minimal interface/classes while implementing.
e.g. when creating POST api, you only need `AddPoi` method, then until that point you only need `AddPoi` until you implement something else, like GET api.

An  exception is when you do something like `GenericRepository`, in that case most basic Repo methods are already implemented for every Entity.

Reply:
Are you saying that we old create a generic interface or even a base class later on, till we have all repo(s) implemented (to know what to be generic)?

Reply:
U're not doing `GenericRepo` so just forget about my latter point. Only implement what's needed is good enough.